packages/contracts/test/contracts/L1/rollup/ChainStorageContainer.spec.ts

```diff
@@ -96,18 +71,14 @@ describe('ChainStorageContainer', () => {
           for (let i = 0; i < len; i++) {
             const value = NON_NULL_BYTES32
             values.push(value)
-            await ChainStorageContainer.connect(otherSigner)['push(bytes32)'](
-              value
-            )
+            await ChainStorageContainer['push(bytes32)'](value)
           }
         })
 
         for (let i = len - 1; i > 0; i -= Math.max(1, len / 4)) {
           it(`should be able to delete everything after and including the ${i}th/st/rd/whatever element`, async () => {
             await expect(
-              ChainStorageContainer.connect(otherSigner)[
-                'deleteElementsAfterInclusive(uint256)'
-              ](i)
+              ChainStorageContainer['deleteElementsAfterInclusive(uint256)'](i)
```

Comment:
Did we modify a formatter/linter rule or something? I didn't think we really had the freedom to decide about stuff like this

Reply:
Hmmm what do you mean by this? What sort of stuff are you referring to?

Reply:
Shouldn't [running](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts/package.json#L34-L37) `eslint --fix .` make the formatting decision for you?

Reply:
Oh it's because I removed `.connect(otherSigner)`

Reply:
Running `yarn lint:fix` worked for me

Reply:
I think its your version of typescript locally

Reply:
that was it yep